src/Benchmarks/project.json

```diff
@@ -1,52 +1,47 @@
 {
-  "version": "1.0.0-*",
-  "compilationOptions": {
+  "version": "1.1.0-*",
+  "buildOptions": {
     "emitEntryPoint": true,
-    "preserveCompilationContext": true
+    "preserveCompilationContext": true,
+    "copyToOutput": {
+      "include": [
+        "appsettings.json",
+        "wwwroot",
+        "Views"
+      ]
+    }
   },
   "dependencies": {
     "Dapper": "1.50.0-*",
-    "Microsoft.EntityFrameworkCore.SqlServer": "1.0.0-*",
-    "Microsoft.EntityFrameworkCore.Tools": {
+    "Microsoft.EntityFrameworkCore.SqlServer": "1.1.0-*",
+    "Microsoft.EntityFrameworkCore.Design": {
       "type": "build",
       "version": "1.0.0-*"
     },
-    "Microsoft.AspNetCore.Mvc": "1.0.0-*",
-    "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*",
+    "Microsoft.AspNetCore.Mvc": "1.1.0-*",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.1.0-*",
     "Microsoft.AspNetCore.Server.WebListener": "0.1.0-*",
-    "Microsoft.AspNetCore.StaticFiles": "1.0.0-*",
-    "Microsoft.Extensions.Options.ConfigurationExtensions": "1.0.0-*",
-    "Microsoft.Extensions.Configuration.Json": "1.0.0-*",
-    "Microsoft.Extensions.Configuration.CommandLine": "1.0.0-*",
-    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0-*"
+    "Microsoft.AspNetCore.StaticFiles": "1.1.0-*",
+    "Microsoft.Extensions.Options.ConfigurationExtensions": "1.1.0-*",
+    "Microsoft.Extensions.Configuration.Json": "1.1.0-*",
+    "Microsoft.Extensions.Configuration.CommandLine": "1.1.0-*",
+    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.1.0-*"
   },
   "frameworks": {
     "netcoreapp1.0": {
-      "imports": [ "portable-net451+win8", "dnxcore50" ],
       "dependencies": {
         "Microsoft.NETCore.App": {
           "version": "1.0.0-*",
           "type": "platform"
         },
-        "System.Runtime.Serialization.Primitives": "4.1.0-*"
+        "System.Runtime.Serialization.Primitives": "4.1.1"
       }
     },
-    "net451": { }
+    "net451": {}
   },
   "tools": {
-    "Microsoft.EntityFrameworkCore.Tools": {
-      "version": "1.0.0-*",
-      "imports": [
-        "dnxcore50",
-        "portable-net452+win81"
-      ]
-    }
+    "Microsoft.EntityFrameworkCore.Tools": "1.0.0-*"
   },
-  "content": [
-    "appsettings.json",
```

Comment:
Were these meant to be copied only to build output? Otherwise need to add them to `publishOptions`.